### ISSUE TITLE
An error occurred attempting to install Democracy3LanguageSelector

### ISSUE THREAD

**Issue by Arkanium77** [dragouf/Democracy3LanguageSelector#4]
[pic](https://i.gyazo.com/e7b645b8f73cc6bb5663baebfb9615a2.png)

text of install.log:

> The following properties have been set:
> Property: [AdminUser] = true {boolean}
> Property: [InstallMode] = HomeSite {string}
> Property: [NTProductType] = 1 {int}
> Property: [ProcessorArchitecture] = AMD64 {string}
> Property: [VersionNT] = 6.3.0 {version}
> Running checks for package 'Microsoft .NET Framework 4.5 (x86 and x64)', phase BuildList
> Reading value 'Version' of registry key 'HKLM\Software\Microsoft\NET Framework Setup\NDP\v4\Full'
> Read string value '4.6.00081'
> Setting value '4.6.00081 {string}' for property 'DotNet45Full_TargetVersion'
> The following properties have been set for package 'Microsoft .NET Framework 4.5 (x86 and x64)':
> Property: [DotNet45Full_TargetVersion] = 4.6.00081 {string}
> Running checks for command 'DotNetFX45\dotNetFx45_Full_x86_x64.exe'
> Result of running operator 'ValueEqualTo' on property 'InstallMode' and value 'HomeSite': true
> Result of checks for command 'DotNetFX45\dotNetFx45_Full_x86_x64.exe' is 'Bypass'
> Running checks for command 'DotNetFX45\dotNetFx45_Full_setup.exe'
> Result of running operator 'ValueNotEqualTo' on property 'InstallMode' and value 'HomeSite': false
> Result of running operator 'VersionGreaterThanOrEqualTo' on property 'DotNet45Full_TargetVersion' and value '4.5.50709': true
> Result of checks for command 'DotNetFX45\dotNetFx45_Full_setup.exe' is 'Bypass' 'Microsoft .NET Framework 4.5 (x86 and x64)' RunCheck result: No Install Needed Launching Application.
> URLDownloadToCacheFile failed with HRESULT '-2146697210'
> Error: An error occurred trying to download 'https://dl.dropboxusercontent.com/u/260255445/Democracy%203/ClickOnce/Democracy3LanguageSelector.application'.

So what's the problem and how to fix it?

**Comment by bancadri** [comment]
I've got the same problem. I think we need a new link to download.

**Comment by dragouf** [comment]
Yes, it was hosted on dropbox and dropbox closed public folder. I have to move it elsewhere but I didn't have time.

**Comment by Arkanium77** [comment]
We'll wait. Can you write in this thread when it'll works?

**Comment by dragouf** [comment]
Yes, sorry I don't have so much time now but soon I will try.

**Comment by Akagilnc** [comment]
anyone still tracks this problem?

**Comment by Akagilnc** [comment]
@dragouf I think it's not a terrible problem. If you need to change the files to a new public address I think I can help you. Then you just need to update the download address in your application.

**Comment by pikacode** [comment]
the same problem

**Comment by XxAlexX15** [comment]
> Yes, sorry I don't have so much time now but soon I will try.

In the end have you done something to solve the problem? Dragouf, please, you said it long ago and since then, it is impossible to play Democracy 3 in a language other than English. Do not you delay more, I do not want to think you're take us for idiots. Thanks for the attention.

**Comment by XxAlexX15** [comment]
Really, I do not know if it's helpful. But, it would be enough if you simply upload the already translated files to any server to be able can, although it's, to apply the translation manually.

**Comment by pikacode** [comment]
Before I die, I'll tell my son: "If this is updated, tell me at my grave!"
